build/repo.props

```diff
@@ -14,6 +14,6 @@
 
   <ItemGroup>
     <DotNetCoreRuntime Include="$(MicrosoftNETCoreApp20PackageVersion)" />
```

Comment:
Remove?

Reply:
I talked it over with @natemcmaster and we decided that due to the complexities of figuring out where to keep and where to delete these things across 50 repos, we're just going to leave them for now. They don't hurt anything, and I've filed aspnet/Universe#1153 to remove them in the future.